index.js

```diff
@@ -240,6 +240,26 @@ const addLessLoader = (loaderOptions = {}) => config => {
   return config;
 };
 
+// Use this helper to add `worker-loader` into webpack loaders.
+const addWorkerLoader = (loaderOptions = {}) => config => {
+  const loaders = config.module.rules.find(rule => Array.isArray(rule.oneOf)).oneOf;
+
+  loaders.splice(loaders.length - 1, 0, {
```

Comment:
What is this splice doing here? Adding to the end of the array the new object?

Reply:
Keep the same behavior with `addLessLoader`.

```javascript
  // Insert less-loader as the penultimate item of loaders (before file-loader)
  loaders.splice(loaders.length - 1, 0, {
    test: lessRegex,
    exclude: lessModuleRegex,
```

Reply:
I'll give an online example later.

Reply:
Oh ok I see, thanks for the explanation!

Reply:
Sorry, I found some error to fix. You are right, the loader shouldn't be added as the penultimate item. This merge should be reverted.

Here is an example:
https://github.com/liyuanqiu/addWorkerLoaderDemo